website/content/docs/agent-and-proxy/agent/caching/persistent-caches/kubernetes.mdx

```diff
@@ -1,21 +1,21 @@
 ---
 layout: docs
-page_title: Kubernetes - Vault Agent Persistent Cache
-description: Kubernetes Persistent Cache for Vault Agent Caching
+page_title: Kubernetes - OpenBao Agent Persistent Cache
+description: Kubernetes Persistent Cache for OpenBao Agent Caching
 ---
 
-# Vault agent kubernetes persistent cache
+# OpenBao agent kubernetes persistent cache
 
-When `kubernetes` is configured for the persistent cache type, Vault Agent will optimize the
+When `kubernetes` is configured for the persistent cache type, OpenBao Agent will optimize the
 persistent cache specifically for Kubernetes. This type of persistent cache requires a Kubernetes
 service account token. The service account token is used during encryption and decryption of the
 persistent cache as an additional integrity check.
 
-The Vault Agent persistent cache file in Kubernetes should only be used for handing off Vault tokens
-and leases between initialization and sidecar Vault Agent containers. This cache file should be shared
-using a memory volume between the Vault Agent containers.
+The OpenBao Agent persistent cache file in Kubernetes should only be used for handing off OpenBao tokens
+and leases between initialization and sidecar OpenBao Agent containers. This cache file should be shared
+using a memory volume between the OpenBao Agent containers.
 
-If the Vault Agent Injector for Kubernetes is being used, the persistent cache is automatically configured
+If the OpenBao Agent Injector for Kubernetes is being used, the persistent cache is automatically configured
```

Comment:
The ID target in the URL (text after the hash sign) is likely based on a section title, and has likely already been updated.  Please check

Reply:
This ID was not renamed. See: https://github.com/JanMa/openbao/blob/sanitize-docs/website/content/docs/platform/k8s/injector/annotations.mdx#L199